src/site/stages/build/drupal/health-care-region.js

```diff
@@ -24,6 +24,64 @@ function sortServices(sortItem) {
     .value();
 }
 
+// Creates the past-events listing pages
+function createPastEventListPages(page, drupalPagePath, files) {
+  const sidebar = page.facilitySidebar;
+  // Events listing page
+  const allEvents = page.pastEvents;
+
+  // store past & current events
+  const pastEventTeasers = {
+    entities: [],
+  };
+
+  // separate current events from past events;
+  _.forEach(allEvents.entities, value => {
+    const eventTeaser = value;
+    const startDate = eventTeaser.fieldDate.startDate;
+    const isPast = moment().diff(startDate, 'days');
+    if (isPast >= 1) {
+      pastEventTeasers.entities.push(eventTeaser);
+    }
+  });
```

Comment:
We don't need lodash for this.

```js
  // separate current events from past events;
  allEvents.entities.forEach(eventTeaser => {
    const startDate = eventTeaser.fieldDate.startDate;
    const isPast = moment().diff(startDate, 'days');
    if (isPast >= 1) {
      pastEventTeasers.entities.push(eventTeaser);
    }
  });
```

Since the for loop only serves as a way of filtering past items out of an array, you could take it even further:

```js
  pastEventTeasers.entities = allEvents.filter(
    eventTeaser => moment().diff(eventTeaser.fieldDate.startDate, 'days') >= 1,
  );
```

Reply:
That is slick, @bkjohnson ! Ty :+1: